flow-report/src/summary/opportunities.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {FunctionComponent} from 'preact';
+import {useMemo} from 'preact/hooks';
+
+import {Util} from '../../../report/renderer/util';
+import {useFlowResult} from '../util';
+
+const MAX_TOP_OPPORTUNITIES = 5;
```

Comment:
WDYT about calling this something other than "opportunities" since that term already has a specific meaning in the performance category? (at least for now).

- Highest Impact Audits (I think this is what we tested with users)
- Top Audits
- Top Improvements
- Top Issues

I'm also thinking we shift the intent here and tone down our promise on them being "Top" if we don't have a lot of confidence yet that we're actually surfacing the highest impact things at first. Examples...

- Common Issues
- Issues At-a-Glance
- Quick Fixes

Reply:
> Highest Impact Audits (I think this is what we tested with users)
> Common Issues

These two are my top picks. "Highest Impact Audits" because we tested it with users, "Common Issues" because it best describes the current algo. If I had to pick one it would be "Common Issues" because it's shorted 🙃